Model/CoXnatProvisionerTarget.php

```diff
@@ -696,27 +707,29 @@ protected function syncProject($coProvisioningTargetData, $provisioningData) {
     //$this->log("coProvisioningTargetData: " . print_r($coProvisioningTargetData, true));
     //$this->log("provisioningData: " . print_r($provisioningData, true));
     // CoService conditions necessary for XNAT project creation:
+    // ['CoService']['name'] not empty
     // ['CoService']['status'] == Active
     // ['CoService']['cou_id'] not empty
     // ['CoService']['co_group_id'] matches a linked provisioning target ['co_group_id']
-    // ['CoService']['identifier_type'] not empty    ?? check this is the same as the provisioner?? 
+    // ['CoService']['identifier_type'] not empty
     // ['CoService']['short_label'] not empty
 
-    $this->log("Groups: provData - TargetData " . print_r($provisioningData['CoService']['co_group_id'] . '+' . $coProvisioningTargetData['CoXnatProvisionerTarget']['co_group_id'], true));
-
     if (!($provisioningData['CoService']['co_group_id'] == $coProvisioningTargetData['CoXnatProvisionerTarget']['co_group_id'])) {
       throw new InvalidArgumentException(_txt('er.service.group.none'));
+      return false;
```

Comment:
I believe adding a `return` after `throw` is redundant because `throw` leaves the function immediately in search of a `catch`. So this `return` is unreachable.

This comment also applies to a few other places in this PR.